<?xml version="1.0"?>

<launch>
    <group ns="dash">
        <node name="dashboard" pkg="ublox" type="ublox_dashboard.py" output="screen">
			<param name="relpostopic" value="/rover/RelPos"/>
            <param name="title" value="Walking Test March 03, 2020"/>
		</node>
    </group>
</launch>